joomla/joomla-cms
# [4.0] Change 'Module Class Suffix' to 'Module Class'

Pull Request for Issue #16894 .

### Summary of Changes
Changes 'Module Class Suffix' to 'Module Class'.


### Testing Instructions
Add class to module using the 'Module Class' field (no space). Check frontend and ensure class has been added correctly.

Eg.
`<div class="card yourClass">`

### Expected result



### Actual result



### Documentation Changes Required
Yes
